lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -194,7 +193,7 @@ describe('LMSFilePicker', () => {
       );
       assert.isTrue(tryAgainButton.exists());
 
-      const errorDetails = wrapper.find(ErrorDisplay);
+      const errorDetails = wrapper.find('ErrorDisplay');
```

Comment:
Was this change needed because `ErrorDisplay` is now being mocked with the new version of `mockImportedComponents` whereas it wasn't before?

Reply:
Yes, I believe so. In any case, it seemed more sensible to use the component's name string than have a dependency in this test on that component module.